sky/engine/public/platform/sky_settings.h

```diff
@@ -17,6 +17,7 @@ struct SkySettings {
   bool start_paused = false;
   bool enable_dart_checked_mode = false;
   bool trace_startup = false;
+  bool complete_timeline = false;
```

Comment:
Yeah, but the name doesn't make sense outside that context.  (I'm not even sure what the name means in the context of the Dart VM.)